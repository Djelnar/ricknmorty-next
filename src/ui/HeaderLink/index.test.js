/* global describe, it, expect */
import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import { HeaderLink } from './index'


describe('HeaderLink', () => {
  it('should render correctly', () => {
    const output = renderer.create(<HeaderLink href="/mock">Mock Content</HeaderLink>).toJSON()

    expect(output).toMatchSnapshot()
  })
})
