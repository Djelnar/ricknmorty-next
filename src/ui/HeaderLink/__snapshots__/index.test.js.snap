// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderLink should render correctly 1`] = `
.c0 {
  display: block;
  color: #ffffff;
  padding-left: 8px;
  padding-right: 8px;
  -webkit-text-decoration: none;
  text-decoration: none;
}

<a
  className="c0"
  href="/mock"
>
  Mock Content
</a>
`;
